ui/src/app/views/homepage/component.tsx

```diff
@@ -453,15 +285,15 @@ export const Homepage = withUserProfile()(class extends React.Component<
           <div style={homepageStyles.singleCard}>
             {accessTasksLoaded ?
               (accessTasksRemaining ?
-                (<WorkbenchAccessTasks eraCommonsLinked={eraCommonsLinked}
-                                       eraCommonsError={eraCommonsError}
-                                       trainingCompleted={trainingCompleted}
-                                       firstVisitTraining={firstVisitTraining}
-                                       betaAccessGranted={betaAccessGranted}/>
+                (<RegistrationPage eraCommonsLinked={eraCommonsLinked}
```

Comment:
Nit: I wouldn't call this `RegistrationPage`, because it isn't actually a top level page? Maybe just `Registration`?

Reply:
I'm going to do `RegistrationDashboard`